README.md

```diff
@@ -120,6 +137,14 @@ Please consider giving a ⭐️ if you like this project!
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+> Give credit to everyone who inspired your codebase.
+
+I would like to thank [Microverse](https://www.microverse.org/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
 ### 📝 License
 
 - This project is MIT lincensed.
```

Comment:
- [ OPTIONAL] It would be nice to provide the link to the LICENSE here. That way contributors can easily know the legal ramifications concerning this project 👍 😄